src/main/java/ch/wisv/areafiftylan/service/TicketService.java

```diff
@@ -13,6 +15,8 @@
 
     Integer getNumberSoldOfType(TicketType type);
 
+    Collection<Ticket> findValidTicketsByOwnerUsername(String username);
```

Comment:
I personally would use a User object as parameter. I feel like it is better to keep data together as much as possible (whilst still relevant ofcourse). It could be the case that later you'd like to do a check in the service on another property of a user (like if they have a profile, no profile means no tickets so you don't have to perform a query.)

Only narrow down to one data point (the username in this case) when performing the query (so in the repository this type of parameter is ok).

I think this an overall better programming style because it gives more flexibility at no cost.